### ISSUE TITLE
Key visitsAggregated does not exist as the array is empty in _profileSummaryVisits.twig

### ISSUE THREAD

**Issue by mattab** [matomo-org/matomo#12579]
Error seen in production:

```
Error: {"message":"Key \"visitsAggregated\" does not exist as the array is empty.","file":"\/plugins\/Live\/templates\/_profileSummaryVisits.twig","line":4}

URL: index.php?date=2018-02-16&module=Live&action=getVisitorProfilePopup&visitorId=8dbccd24ae6a5b65&idSite=1&period=month
```

**Comment by sgiehl** [comment]
Shouldn't that be fixed with #12396. Is that server running latest beta?

**Comment by mattab** [comment]
@sgiehl it's probably fixed :+1: so I will close it for now and re-open later if it occurs again with the latest version.
